packages/core/src/sdk/reviews/useProductReviews.ts

```diff
@@ -0,0 +1,101 @@
+import { useState, useCallback, useEffect } from 'react'
+import type {
+  ClientProductReviewsQuery,
+  StoreProductListReviewsSort,
+} from '@generated/graphql'
+import { useProductReviewsQuery } from './useProductReviewsQuery'
+import type { FilterProductListReview } from 'src/components/reviews/FilterProductReviews/FilterProductReviews'
+
+type ProductReview = ClientProductReviewsQuery['reviews']['data'][number]
+
+const PAGE_SIZE = 6
```

Comment:
```suggestion
const REVIEWS_PER_PAGE = 6
```

suggestion: Rename to be more descriptive

Reply:
Done in https://github.com/vtex/faststore/pull/2724/commits/d6029af45e10085552ef3bcffa44dea65e3ebd66